apps/client/src/pages/confeti/components/performance-detail.tsx

```diff
@@ -1,17 +1,24 @@
 import * as styles from '@pages/confeti/components/performance-detail.css';
 
-const PerformanceDetail = () => {
+interface PerformanceDetailProps {
+  isExpanded: boolean;
+}
+
+const PerformanceDetail = ({ isExpanded }: PerformanceDetailProps) => {
   return (
-    <>
-      <div className={styles.container}>
-        <div className={styles.wrapper}>
-          <div className={styles.title}>공연 상세</div>
-          <img src="https://ticketimage.interpark.com/Play/image/etc/24/24016171-02.jpg" />
-        </div>
-        {/* 추후 더보기 버튼 컴포넌트 추가 예정 */}
-        <button></button>
+    <div
+      className={`${styles.container} ${
+        isExpanded ? styles.expanded : styles.collapsed
+      }`}
+    >
+      <div className={styles.wrapper}>
+        <div className={styles.title}>공연 상세</div>
```

Comment:
```suggestion
        <h2 className={styles.title}>공연 상세</h2>
```
위에 리뷰남긴 내용과 같은 이유로 h2로 수정해주면 좋아보여요 !